src/libs/actions/Policy.ts

```diff
@@ -2779,7 +2779,11 @@ function navigateWhenEnableFeature(policyID: string, featureRoute: Route) {
         return;
     }
 
-    Navigation.navigate(featureRoute);
+    new Promise<void>((resolve) => {
+        resolve();
+    }).then(() => {
```

Comment:
Why do we need this promise?

Reply:
@luacmartins here is more details about that https://github.com/Expensify/App/issues/37785#issuecomment-1989056726

Reply:
Thanks for the context! I think we should leave a comment about that then